Makefile

```diff
@@ -325,9 +325,11 @@ e2e-deploy-ratify: e2e-notaryv2-setup e2e-cosign-setup e2e-licensechecker-setup
 	--set-file provider.tls.key=${CERT_DIR}/server.key \
 	--set provider.tls.cabundle="$(shell cat ${CERT_DIR}/ca.crt | base64 | tr -d '\n')" \
 	--set ratifyTestCert="$$(cat ~/.config/notation/localkeys/ratify-bats-test.crt)" \
+	--set cosign.enabled=true \
 	--set cosign.key="$$(cat .staging/cosign/cosign.pub)" \
 	--set oras.useHttp=true
 
+	kubectl delete verifiers.config.ratify.deislabs.io/verifier-cosign 
```

Comment:
we want to install ratify with cosign enabled so the crt is mounted, but we will disable cosign so e2e test runs in a clean state.

Reply:
maybe we can have an issue for tracking the refactoring?

Reply:
agreed. I think at this point we may need to overhaul cosign so we can properly support it.

Reply:
@binbin-li , do you mean have a issue to track cosign refactoring? we have an issue here https://github.com/deislabs/ratify/issues/194 to refactor cosign into its own store